text/0133-dynamic-component-creation.md

```diff
@@ -202,7 +202,7 @@ _Note in the case of lazy loading, the component author is responsible for resol
 
 #### Selecting the dynamic component
 
-To select the dynamic component, the actual custom element must be selected once it has been rendered to the DOM by using the [`lwc:ref`](https://rfcs.lwc.dev/rfcs/lwc/0000-refs) directive or through another attribute assigned to the component such as a class name.
+To select the dynamic component, the actual custom element must be selected once it has been rendered to the DOM by using the [`lwc:ref`](https://github.com/salesforce/lwc-rfcs/blob/ccf0187bec2165d22c5212ac9839da457dbd5dde/text/0000-refs.md) directive or through another attribute assigned to the component such as a class name.
```

Comment:
This should be `master` instead of a particular commit. Same for some of the other links in this PR.

Reply:
The issue is that these particular rfc's are no longer available on main branch.That's why we reference to that particular commit where they exist instead of master branch.

Reply:
The `refs` file is now located at https://github.com/salesforce/lwc-rfcs/blob/master/text/0122-refs.md

Reply:
Oh.I will update the references to this then.Thanks

Reply:
Done